src/Main.hs

```diff
@@ -11,17 +11,17 @@
 -- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
+--
 module Main where
 
 import Vector
+import Color
+import Camera
+import Utils
```

Comment:
Given that this is now a fairly generic module name, might be good to explicitly state what functions we're importing. Just makes it easier to see where otherwise in-obvious functions have come from